### DISCUSSION TITLE
One SIEM Two Subnets, is this possible?

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7710] [Unsupported Versions] [answered]
Question for the Security Onion Professionals,

Is it possible to sense two different domains with same IP schema and maintain log integrity to be able to differentiate between the two domains? Or will Sec Onion de-dup that data?

**Comment by dougburks** [comment, marked as answer]
You should be able to do this. You will probably want a separate sensor for each subnet so that you can filter based on the sensor that saw the traffic.

**Reply by ColeVan** [reply]
I would suspect that beat data from two different domains that use same subnets wouldn't be an issue either? Unless host names are exactly the same.

**Reply by dougburks** [reply]
That sounds correct.
